receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -950,3 +966,28 @@ metrics:
       value_type: int
     attributes: [document_state, index_aggregation_type]
     enabled: false
+  elasticsearch.process.cpu.usage:
+    description: CPU usage in percent, or -1 if not known at the time the stats are computed.
+    unit: '%'
```

Comment:
We should convert to decimal between 0 and 1, since this is the guideline for usage metrics.